services/web/server/src/simcore_service_webserver/computation_api.py

```diff
@@ -65,13 +65,40 @@ async def _get_node_details(
         app, node_key, node_version
     )
     if not node_details:
-        log.error("Error could not find service %s:%s", node_key, node_version)
+        log.error(
+            "Error (while getting node details) could not find service %s:%s",
+            node_key,
+            node_version,
+        )
         raise web_exceptions.HTTPNotFound(
             reason=f"details of service {node_key}:{node_version} could not be found"
         )
     return node_details
 
 
+async def _get_node_extras(
+    node_key: str, node_version: str, app: web.Application
+) -> Dict:
+    """Returns the service_extras if possible otherwise None"""
+    if any(
+        key in node_key
+        for key in ("file-picker", "frontend/nodes-group", "StimulationSelectivity")
```

Comment:
so looking forward to not having these hard-coded exceptions... can you please put then in a const variable e.g. like ``EXCLUDE_NODE_KEYNAMES`` at the top of the file ... this way we do forget it!

Reply:
Yes the suggested changes to `async_is_gpu_node`, it will take a bit of time as I would need to adjust the mocks in the tests.

Reply:
so as stated above, use the node_class attribute to filter out. this way you should not need the service names here.

Reply:
Yes, I have finally figured that out.